.gitignore

```diff
@@ -34,4 +34,4 @@ _dev/apps/ui/.jest-test-results.json
 _dev/apps/ui/storybook-static
 .vscode/launch.json
 .pnpm-store
-views/js/fetchVerificationTag.js*
+views/js/*
```

Comment:
This line is too aggressive in my opinion because some files are currently tracked in this folder.
More files could be tracked in the future as well, and with this line there would never appear in the files list ready to be staged.

I would rather manually set the list of files to ignore.